test/image/mocks/multicategory_series.json

```diff
@@ -0,0 +1,1618 @@
+                240,
+                195,
+                155,
+                120,
+                100,
+                80,
+                75,
+                90,
+                115
+            ],
+            "name": "B - 4",
+            "yaxis": "y",
+            "legendgroup": "B - 4",
+            "hoverinfo": "none",
+            "showlegend": false
+        },
+        {
+            "type": "scatter",
+            "mode": "lines",
+            "line": {
+                "dash": "dot",
+                "shape": "spline",
+                "color": "#737373"
+            },
+            "hovertemplate": "",
+            "x": [
+                [
+                    "Low",
+                    "Low",
+                    "Low",
+                    "Low",
+                    "Low",
+                    "Low",
+                    "Low",
+                    "Low",
+                    "Low",
+                    "Low",
+                    "Low",
+                    "Low"
+                ],
+                [
+                    0,
+                    1,
+                    2,
+                    3,
+                    4,
+                    5,
+                    6,
+                    7,
+                    8,
+                    9,
+                    10,
+                    11
+                ]
+            ],
+            "y": [
+                295,
+                300,
+                275,
+                240,
+                195,
+                155,
+                120,
+                100,
+                80,
+                75,
+                90,
+                115
+            ],
+            "name": "A - 4",
+            "yaxis": "y",
+            "legendgroup": "A - 4",
+            "hoverinfo": "none",
+            "showlegend": false
+        }
+    ],
+    "layout": {
+        "autosize": true,
+        "margin": {
+            "t": 40
+        },
+        "legend": {
+            "x": 0.5,
+            "y": "auto",
+            "xanchor": "center",
+            "yanchor": "top",
+            "orientation": "h",
+            "bgcolor": "transparent",
+            "font": {
+                "size": 10
+            }
+        },
+        "hovermode": "x unified",
+        "title": {
+            "text": "Multicategory Mock with Tickmode 'sync'"
+        },
+        "xaxis": {
+            "visible": false,
+            "range": [
+                0,
+                42
+            ],
+            "autorange": true,
+            "type": "multicategory"
+        },
+        "yaxis": {
+
+            "range": [
+                62.5,
+                312.5
+            ],
+            "anchor": "x",
+            "side": "right",
+            "type": "linear"
+        },
+        "yaxis2": {
+            "range": [
+                6150,
+                7451.09375
+            ],
+            "type": "linear",
+            "side": "left",
+            "tickmode": "sync",
```

Comment:
*sync* `tickmode` would be implemented in #6356. Let's not add it here which might be confusing. 
This is a pretty interesting mock to have. But it may be better to simplify it and add a jasmine test in [hover_label_test](https://github.com/plotly/plotly.js/blob/master/test/jasmine/tests/hover_label_test.js) instead.